rclcpp/include/rclcpp/exceptions/exceptions.hpp

```diff
@@ -100,6 +100,14 @@ class InvalidServiceNameError : public NameValidationError
   {}
 };
 
+class UnimplementedError : public std::runtime_error
+{
+public:
+  UnimplementedError()
+  : std::runtime_error("This code is unimplemented.") {}
+  explicit UnimplementedError(const std::string & msg)
+  : std::runtime_error(msg) {}
+};
```

Comment:
add a line?

```suggestion
};

```

Reply:
Thanks, fixed. Surprised this sort of thing isn't caught by the linter.